test/CMakeLists.txt

```diff
@@ -31,6 +31,14 @@ ADD_FLUID_TEST(test_seq_event_queue_remove)
 ADD_FLUID_TEST(test_jack_obtaining_synth)
 ADD_FLUID_TEST(test_utf8_open)
 
+if ( GLIB_SUPPORT AND GLib2_VERSION VERSION_GREATER_EQUAL 2.33.12 )
```

Comment:
Can you pls. add a comment to the code that explains why this version requirement is needed?